examples/logo_interpreter.rs

```diff
@@ -0,0 +1,81 @@
+//! To run a LOGO program, run:
+//!     cargo run --example logo_interpreter -- my_logo_program.txt
+
+extern crate turtle;
+
+use turtle::Turtle;
+use std::env;
+use std::fs::File;
+use std::io::{self, BufRead, BufReader};
+
+fn main() {
+    let mut turtle = Turtle::new();
+
+    let args: Vec<String> = env::args().collect();
+
+    if args.len() != 2 {
+        println!(
+            "Interactive LOGO Mode.\nType the commands in stdin.\nPress '^C' (cmd/ctrl+C) to exit"
+        );
```

Comment:
I don't think this this `println` needs to be spread across several lines like this. Please just put it all on one line.